account_claims.go

```diff
@@ -165,3 +176,18 @@ func (a *AccountClaims) ExpectedPrefixes() []nkeys.PrefixByte {
 func (a *AccountClaims) Claims() *ClaimsData {
 	return &a.ClaimsData
 }
+
+func (a *AccountClaims) DidSign(op Claims) bool {
```

Comment:
just to be clear (maybe with doc) this is post decode so the signature is already validated. You depend on that below.

Reply:
Not quite sure how that helps. If the claim (the argument) is not decoded, it won't have the signer so the test will always fail. If it is decoded then as long as the instance exists (whether created or read from JWT) if the issuer key appears in the signing keys it is good to go. Now the only thing that could be helpful is to verify that the signing keys that are added are validated on add  - currently, it is a validation issue, but really it should be tested that they are good on add as well.

Part of the reason I wanted to remove the extras is I think we should make this part of the interface, so a claim can verify if it's signed by another.

Reply:
if i just make two claims i can't know that this method would be true, since i haven't really "signed" anything. I like the method name, but without any doc it is basically "is the issuer" not "did sign" since we don't know the claims were ever signed (it is a minor thing but i think worth putting in the doc for that method (which we need anyway for a public method)

Reply:
Note this is code that mirrors your code for the operators.

Reply:
some one should have caught that in the review